docs/guide/image_editor.md

```diff
@@ -0,0 +1,82 @@
+# Configuring the Image Editor
+
+## Introduction
+
+When a Content Item contains Fields of the [ezimage](../api/field_type_reference.md#image-field-type) or [ezimageasset](../api/field_type_reference.md#imageasset-field-type) type, users can perform basic image editing functions with the Image Editor.
```

Comment:
```suggestion
When a Content item contains Fields of the [ezimage](../api/field_type_reference.md#image-field-type) or [ezimageasset](../api/field_type_reference.md#imageasset-field-type) type, users can perform basic image editing functions with the Image Editor.
```

Reply:
I think it is worth mentioning that its not working on DAM assets atm (button would be disabled/invisible).